### ISSUE TITLE
fit_class_random_forest: process should return training and validation accuracy

### ISSUE THREAD

**Issue by JeroenVerstraelen** [Open-EO/openeo-processes#354]
**Describe the issue:**
The fit_class_random_forest process requires a training parameter which a back-end uses to split the data in a training and validation set. However, there is no description for what should be done with this validation set.

**Proposed solution:**
Return the training and validation accuracy perhaps as part of the STAC ml-model extension. This allows the user to fine-tune the hyper-parameters based on these values.

**Comment by m-mohr** [comment]
Yes, I think we should create a PR for the ml-model extension to support this. Once that is done we can clarify this in the process, too.

**Comment by m-mohr** [comment]
@JeroenVerstraelen @jdries Please review PR #351
